src/components/OrganizationCard.js

```diff
@@ -10,7 +10,13 @@ class OrganizationCard extends Component {
 
     let distance;
     if(this.props.haveCoords){
-      distance = <p>Distance from your Location: {getDistance({coordinates: this.props.organization.coordinates}, this.props.currentPos ).toPrecision(4)} miles</p>
+      if( this.props.organization.coordinates !== undefined){
```

Comment:
I think this was fixed in https://github.com/codeforboston/communityconnect/pull/55